.github/workflows/build.yml

```diff
@@ -42,8 +42,8 @@ jobs:
           NODE_AUTH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       - name: Lint
         run: npm run lint
-      - name: Test
-        run: npm test
+      #- name: Test
+      #  run: npm test
```

Comment:
Easier to just change https://github.com/podaac/swodlr-user-notify/blob/3b9c137582d9cce0c836c5e906993f46d1ecb801/package.json#L8 to exit 0 instead of commenting this out